massa-consensus-worker/src/state/verifications.rs

```diff
@@ -57,7 +80,12 @@ impl ConsensusState {
         stored_block: SecureShareBlock,
         current_slot: Option<Slot>,
     ) -> Result<Option<BlockInfos>, ConsensusError> {
-        match self.check_header(&block_id, &stored_block.content.header, current_slot, self)? {
+        let header_outcome =
+            self.check_header(&block_id, &stored_block.content.header, current_slot, self)?;
+        if self.is_multistake(&stored_block.content.header) {
+            return Ok(None);
```

Comment:
problem: if we really want (wishlist) we should accept it even if it is multistaked